### PR TITLE
Rename wikiclass to articlequality

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T181820

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def read(fname):
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'wikiclass', u'Wiki-Class Documentation',
+    ('index', 'articlequality', u'Wiki-Class Documentation',
```
Comment (halfak) [inline doc/conf.py]: "Article Quality Documentation"

**Comment by halfak** [comment]
Just one minor note.  Otherwise, it looks great.

File: README.md [hunk 2]
```diff
@@ -5,18 +5,18 @@ This library provides a set of utilities for performing automatic detection of a
 **Compatible with Python 3.x only.**  Sorry.
 
 * **Install:** ``pip install wikiclass``
-* **Models:** https://github.com/wiki-ai/wikiclass/tree/master/models
+* **Models:** https://github.com/wiki-ai/articlequality/tree/master/models
 * **Documentation:** http://pythonhosted.org/wikiclass
```
Comment (halfak) [inline README.md]: http://pythonhosted.org/articlequality (we'll need to upload docs soon after the merge)

Comment (Ladsgroup) [reply README.md]: 404s for me, that's why I thought we should do it in several steps :D

**Pull request merged**